flink-tests/src/test/java/org/apache/flink/api/datastream/DataStreamBatchExecutionITCase.java

```diff
@@ -127,17 +130,17 @@ public void batchSumSingleResultPerKey() throws Exception {
 	}
 
 	private StreamExecutionEnvironment getExecutionEnvironment() {
-		final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 
 		Configuration config = new Configuration();
 		config.set(ExecutionOptions.RUNTIME_MODE, RuntimeExecutionMode.BATCH);
-		env.configure(config, DataStreamBatchExecutionITCase.class.getClassLoader());
+		config.set(CoreOptions.DEFAULT_PARALLELISM, 1);
```

Comment:
So you prefer `config.set(CoreOptions.DEFAULT_PARALLELISM, 1);` to `env.setParallelism(1)`? 😅

Reply:
Not necessarily. It is more to verify that we can pass parallelism that way as well. I can revert.